R/rank_regression.R

```diff
@@ -321,7 +322,11 @@ rank_regression_ <- function(cdf_estimation,
 
   ## Parameters:
   dist_params <- stats::coef(rr)
-  names(dist_params) <- c("mu", "sigma")
+  if (distribution == "exponential") {
+    names(dist_params) <- "sigma"
```

Comment:
The code in `rank_regression_()` is generalized. Since 'sigma' (scale parameter) exists for every distribution, the computation of confidence intervals in `conf_HC()` is based on named quantities ('sigma' and 'mu'). The names of the distribution-dependent parameters are set after general computations.